erpnext/selling/report/pending_so_items_for_purchase_request/pending_so_items_for_purchase_request.py

```diff
@@ -139,19 +144,57 @@ def get_data():
 
 		# check for pending sales order
 		if cint(so.net_qty) > cint(materials_request.get('qty')):
-			so_record = {
-				"item_code": so.item_code,
-				"item_name": so.item_name,
-				"description": so.description,
-				"sales_order_no": so.name,
-				"date": so.transaction_date,
-				"material_request": ','.join(materials_request.get('material_requests', [])),
-				"customer": so.customer,
-				"territory": so.territory,
-				"so_qty": so.net_qty,
-				"requested_qty": cint(materials_request.get('qty')),
-				"pending_qty": so.net_qty - cint(materials_request.get('qty')),
-				"company": so.company
-			}
-			pending_so.append(so_record)
-	return pending_so
\ No newline at end of file
+
+			if so.item_code not in item_with_product_bundle:
+				so_record = {
+					"item_code": so.item_code,
+					"item_name": so.item_name,
+					"description": so.description,
+					"sales_order_no": so.name,
+					"date": so.transaction_date,
+					"material_request": ','.join(materials_request.get('material_requests', [])),
+					"customer": so.customer,
+					"territory": so.territory,
+					"so_qty": so.net_qty,
+					"requested_qty": cint(materials_request.get('qty')),
+					"pending_qty": so.net_qty - cint(materials_request.get('qty')),
+					"company": so.company
+				}
+				pending_so.append(so_record)
+			else:
+				for item in packed_items:
```

Comment:
This code will run every time for the product bundle. For example if sales order has 4 items out of which 2 are bundle items then line no 165 will be called two times.

Sales Order
<img width="1183" alt="Screenshot 2019-07-20 at 11 36 59 PM" src="https://user-images.githubusercontent.com/8780500/61582364-9c9be180-ab47-11e9-97db-3ba1181c1b3b.png">


Product Bundle

<img width="1213" alt="Screenshot 2019-07-20 at 11 37 12 PM" src="https://user-images.githubusercontent.com/8780500/61582358-81c96d00-ab47-11e9-9529-1351fa0398e0.png">


Report

<img width="747" alt="Screenshot 2019-07-20 at 11 36 39 PM" src="https://user-images.githubusercontent.com/8780500/61582371-b0474800-ab47-11e9-9557-178b11569b6a.png">